tests/routes/series.test.js

```diff
@@ -59,7 +63,9 @@ describe('Series', () => {
                 vintage_date: 'dates'
             };
             series.getObservationsForSeries(125, params);
-            expect(api.get).to.have.been.calledWith('series/observations?api_key=testkey&file_type=json&series_id=125&realtime_start=2000-10-11&realtime_end=2000-10-15&limit=100&offset=10&sort_order=asc&observation_start=1960-03-10&observation_end=1965-03-10&units=undefined&frequency=10&aggregation_method=avg&output_type=xml');
```

Comment:
`units=undefined`
Not sure why I didn't see this before, but this was the issue